lib/web/users_test.go

```diff
@@ -0,0 +1,44 @@
+package web
+
+import (
+	"testing"
+
+	"github.com/gravitational/trace"
+	"gopkg.in/check.v1"
+)
+
+type UsersSuite struct{}
+
+var _ = check.Suite(&UsersSuite{})
+
+func TestUsers(t *testing.T) { check.TestingT(t) }
```

Comment:
This will cause all `lib/web` tests to run, not just this suite.
`apiserver_test.go` already triggers `check.TestingT`.
Remove this func please

This is one of our pains with gocheck :(

Reply:
removed